aiohue/sensors.py

```diff
@@ -110,6 +117,15 @@ def state(self):
     def config(self):
         return self.raw["config"]
 
+    def generic_process_update_event(self, update, state):
+        if "power_state" in update:
+            state["battery"] = update["power_state"]["battery_level"]
+
+        state["lastupdated"] = datetime.utcnow().replace(microsecond=0).isoformat()
+        self.last_event = update["type"]
```

Comment:
I suggest we store the whole event. We never know what use case one comes up with in the future.
```suggestion
        self.last_event = update
```